From cebbe0cf0334f5ca92aadf771e76105e1c1cfec5 Mon Sep 17 00:00:00 2001
From: KKY <kkyrkwhr@naver.com>
Date: Fri, 4 Aug 2017 13:53:43 +0900
Subject: [PATCH] =?UTF-8?q?=EB=A7=88=EC=A7=80=EB=A7=89?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 asd.html | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 asd.html

diff --git a/asd.html b/asd.html
new file mode 100644
index 0000000..9dafd79
--- /dev/null
+++ b/asd.html
@@ -0,0 +1 @@
+asdasdsad
\ No newline at end of file
-- 
2.12.2.windows.2

